atlas/templates/Makefile.common.gotmpl

```diff
@@ -91,11 +93,12 @@ protobuf-atlas:
 
 .PHONY vendor: vendor-atlas
 vendor-atlas:
-	@dep ensure -vendor-only
-
-.PHONY vendor-update: vendor-update-atlas
-vendor-update-atlas:
-	@dep ensure
+ifneq ($(shell test -e $(GO_MOD) && echo -n y),y)
+	@go mod init
```

Comment:
As I am understand we call ```go mod init``` in process of application bootstrapping, due to this I don't think command is required